public/china/index.html

```diff
@@ -4,41 +4,64 @@
   <title>China</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <!-- <link rel="stylesheet" href="../stylesheets/style.css"> -->
+  <link rel="stylesheet" href="../stylesheets/style.css">
 </head>
 <body>
   <header id="header">
-    <nav id="navigation">
-      <ul>
-        <li><a href="../">Home</a></li>
-        <li><a href="../china/">China</a></li>
-        <li><a href="../japan/">Japan</a></li>
-        <li><a href="../malaysia/">Malaysia</a></li>
-        <li><a href="../singapore/">Singapore</a></li>
-        <li><a href="../south_korea/">South Korea</a></li>
-        <li><a href="../thailand/">Thailand</a></li>
-      </ul>
+    <!-- Sidebar -->
+    <nav class="sidebar barmenu topheader" id="mySidebar">
+      <a onclick="close_menu()" class="button">Close Menu</a>
+      <a href="#" class="button">Feedback</a>
+      <a href="#" class="button">About Us</a>
     </nav>
+
+    <!-- Top -->
+    <div class="fixedtop">
+      <div class="topheader">
+        <div class="hambutton" onclick="open_menu()">☰</div>
+        <div class="logo"><a href="../index.html">Travelbriefing</a></div>
+      </div>
+
+      <div class="countrynavi" id="navigation">
```

Comment:
Why do we need a div here? Can we just add that class and id to the <nav> and style from there?